server/server.go

```diff
@@ -109,16 +109,16 @@ func (r *rodeServer) batchCreateGenericResources(ctx context.Context, occurrence
 
 	mgetBody, _ := encodeRequest(&esutil.EsMultiGetRequest{IDs: resourceNames})
 
-	response, err := r.esClient.Mget(mgetBody, r.esClient.Mget.WithContext(ctx), r.esClient.Mget.WithIndex(rodeElasticsearchGenericResourcesIndex))
+	res, err := r.esClient.Mget(mgetBody, r.esClient.Mget.WithContext(ctx), r.esClient.Mget.WithIndex(rodeElasticsearchGenericResourcesIndex))
 	if err != nil {
-		log.Error("failed to mget documents", zap.Error(err))
 		return err
 	}
-	if response.IsError() {
-		return fmt.Errorf("unexpected status code from mget request: %d", response.StatusCode)
+	if res.IsError() {
+		return fmt.Errorf("unexpected response from elasticsearch: %s [%d]", res.String(), res.StatusCode)
```

Comment:
The response's string method includes the status code:
```go
r := esapi.Response{
    StatusCode: 200,
}
fmt.Println("res", r.String()) // prints res [200 OK]
```

```suggestion
		return fmt.Errorf("unexpected response from elasticsearch: %s", res.String())
```